packages/fuel-gauge/src/void.test.ts

```diff
@@ -0,0 +1,49 @@
+import type { DeployContractConfig } from 'fuels/test-utils';
+import { launchTestNode } from 'fuels/test-utils';
+
+import { VoidAbi__factory } from '../test/typegen';
+import type { NativeEnumInput } from '../test/typegen/contracts/VoidAbi';
+import VoidAbiHex from '../test/typegen/contracts/VoidAbi.hex';
+import type { Option } from '../test/typegen/contracts/common';
+
+/**
+ * @group node
+ */
+describe('Void Tests', () => {
+  const contractsConfigs: DeployContractConfig[] = [
```

Comment:
```suggestion
  const contractsConfigs = [
```

Breaks the type inference